config/foreman.migrations/20240322170637_reset_puppet_server_java_bin.rb

```diff
@@ -0,0 +1,3 @@
+if answers['puppet'].is_a?(Hash) && answers['puppet']['server_jvm_java_bin']
```

Comment:
should this be
```
if answers['puppet'].is_a?(Hash) && answers['puppet']['server_jvm_java_bin'] == '/usr/bin/java'
```

Reply:
Oops. Still doesn't explain why AlmaLinux didn't work

Reply:
Yeah, I just went looking and thought "oh, wait" :)

Reply:
https://github.com/theforeman/foreman-installer/pull/923